client/lib/i18n-utils/switch-locale.js

```diff
@@ -237,8 +246,8 @@ function switchWebpackCSS( isRTL ) {
  * Loads a CSS stylesheet into the page.
  *
  * @param {string} cssUrl URL of a CSS stylesheet to be loaded into the page
- * @param {Element} currentLink an existing <link> DOM element before which we want to insert the new one
- * @returns {Promise<String>} the new <link> DOM element after the CSS has been loaded
+ * @param {window.HTMLElement} currentLink an existing <link> DOM element before which we want to insert the new one
```

Comment:
FWIW `eslint` complained about using `Element` here. I think this is a valid way to make eslint happy but please check.

Reply:
`window.HTMLElement` works, but as a long term (probably aspirational 🙂) goal, it would be nice if JSDoc accepted TypeScript types. In this case, TypeScript has a built-in `Element` type that can be referenced directly with no prefix.

Reply:
Well, `eslint` wouldn't accept `Element` here. What to do?

Reply:
We'll need `window.Element` or `window.HTMLElement` for now, I don't see anything better.